tests/meta-mender-raspberrypi3-ci/recipes-mender/mender-qa/files/rpi/activate-test-image

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+set -e
+
+MOUNT_DEV=${MOUNT_DEV:-/dev/mmcblk0p3}
+UENV_MOUNT=/mnt
+
+case "$1" in
+    status)
+        if [ -e $UENV_FILE ]; then
+            val=$(grep -e "uenvcmd=" $UENV_FILE 2>/dev/null)
+            echo "test image is disabled"
+            disabled=1
+        fi
+        if [ -z "$disabled" ]; then
+            echo "tests image is enabled"
+        fi
+        ;;
+    off)
+        echo "deactivating test image"
+        echo "dwc_otg.lpm_enable=0 console=serial0,115200 rootfstype=ext4 rootwait root=/dev/sda2" > /uboot/cmdline.txt
+    reboot
+        ;;
+    on|'')
+        echo "activating test image"
+        mount $MOUNT_DEV $UENV_MOUNTD && \
+            echo "dwc_otg.lpm_enable=0 console=serial0,115200 rootfstype=ext4 rootwait root=${mender_kernel_root}" > /mnt/uboot/cmdline.txt && \
```

Comment:
One question: Is it possible to delete the file, instead of changing it? The reason is that we're not truly testing the Mender boot command line if this file is always in effect.